lib/matplotlib/mathtext.py

```diff
@@ -3399,26 +3398,23 @@ def to_mask(self, texstr, dpi=120, fontsize=14):
         return x, depth
 
     def to_rgba(self, texstr, color='black', dpi=120, fontsize=14):
-        """
-        *texstr*
-            A valid mathtext string, e.g., r'IQ: $\\sigma_i=15$'
-
-        *color*
-            Any matplotlib color argument
-
-        *dpi*
-            The dots-per-inch to render the text
-
-        *fontsize*
+        r"""
+        Parameters
+        ----------
+        texstr : str
+            A valid mathtext string, e.g., r'IQ: $\sigma_i=15$'.
+        color : color
+        dpi : float
+            The dots-per-inch setting used to render the text.
+        fontsize : int
             The font size in points
 
-        Returns a tuple (*array*, *depth*)
-
-          - *array* is an NxM uint8 alpha ubyte mask array of
-            rasterized tex.
-
-          - depth is the offset of the baseline from the bottom of the
-            image in pixels.
+        Returns
+        -------
+        array : (M, N, 4) RGBA alpha
```

Comment:
This is quite opaque. Maybe:
~~~
array : (M, N, 4) array
    RGBA color values of the TeX expression rastered on an (M, N) grid.
~~~

Reply:
edited to "RGBA color values of rasterized tex, colorized with *color*.".  If this + (M, N, 4) doesn't explain to the user what the array layout is, they likely need an explanation of the RGBA format and that's not the point of this docstring.